### PR TITLE
[RNMobile] Gallery block crash betafix

### LINKED ISSUE #3510: The editor Crashes when adding a Caption to a Gallery block
**Describe the bug**
The editor Crashes when adding a Caption to a Gallery block

**To Reproduce**
Follow the steps on [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/gallery.md#tc003) or:
1. Add a Gallery block
2. Add some images from the device or media library
3. Tap on the gallery caption
4. **Notice** the the editor crashes

The **Crash** occurs both on **Android and iOS**

**Expected behavior**
The user is able to add a gallery caption.

**Screenshots**

https://user-images.githubusercontent.com/304044/118486426-463b6200-b722-11eb-9ae9-7b613ea62e17.mp4

**Smartphone (please complete the following information):**
 - Device: Pixel 2 XL, iPhone SE
 - OS: Android 11.0, iOS 14.4
 - Version Gutenberg Mobile [1.53.0](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3506)

### PR DESCRIPTION
**Pull request by AmandaRiu**
## Description

Fixes [a crash in the Image Gallery block when adding a caption](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3510) by reverting [this commit](https://github.com/WordPress/gutenberg/commit/9a6e3fcbc927791d7fb70f35ba88da1ef2d83acb). 

## How has this been tested?

1. Add a Gallery block
2. Add some images from the device or media library
3. Tap on the gallery caption
4. The app should not crash. 

## Types of changes

Bug fix. 

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

File: packages/react-native-aztec/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/react-native-aztec",
-	"version": "1.52.2",
+	"version": "1.53.0",
```
Comment (dcalhoun) [inline packages/react-native-aztec/package.json]: This change and a few others in the PR I believe are from the release script run for 1.53.0. It is a little odd that we include these changes, but I believe their inclusion originates from the fact that this PR branched from the `rnmobile/1.53.0` tag. I don't believe it is an issue we have to worry about, as the the changes in this PR will be squashed and release script changes will likely "disappear," as they already exist on `trunk`.

**Review by dcalhoun — APPROVED** [review]
LGTM. I tested the changes on an iPhone 11 simulator and a Samsung Galaxy S20. The app no longer crashes when editing the Gallery caption.

File: packages/react-native-editor/CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,10 @@ For each user feature we should also add a importance categorization label  to i
 -   [*] The BottomSheet Cell component now supports the help prop so that a hint can be supplied to all Cell based components. [#30885]
 
 
+## 1.52.2
+
+-   [*] Disabled featured image banner on iOS. [#31681]
+
```
Comment (dcalhoun) [inline packages/react-native-editor/CHANGELOG.md]: Looks like we may have erroneously duplicated the 1.52.2 change log entry.
```suggestion
```

**Comment by hypest** [comment]
Headsup that @ellatrix 's help we took a different approach to only revert the part that's breaking the native mobile side, [here's the PR](https://github.com/WordPress/gutenberg/pull/32029) so, we can probably close this one.

**Comment by fluiddot** [comment]
> Headsup that @ellatrix 's help we took a different approach to only revert the part that's breaking the native mobile side, [here's the PR](https://github.com/WordPress/gutenberg/pull/32029) so, we can probably close this one.

@AmandaRiu the issue has been already addressed in these PRs ([Gutenberg](https://github.com/WordPress/gutenberg/pull/32029), [Gutenberg-Mobile](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3534)) so feel free to close this PR in favor of them.

**Comment by AmandaRiu** [comment]
Fixed in this [PR](https://github.com/WordPress/gutenberg/pull/32029). Closing.

**Pull request closed without merging**